packages/react-app/src/App.jsx

```diff
@@ -113,67 +210,58 @@ function App({ mainnet, localProvider, appChainId }) {
       logoutOfWeb3Modal()
     })
 
-    console.log({ injectedProvider })
-    setTabValue(1)
+    // console.log({ injectedProvider })
+    setTabValue(prev => prev)
```

Comment:
is it being used?

Reply:
This is no longer in use and I have removed it. Thanks